### ISSUE TITLE
valid-jsdoc doesn't recoginze JSDoc namepaths

### ISSUE THREAD

**Issue by asapach** [eslint/eslint#1442]
When using [namepaths](http://usejsdoc.org/about-namepaths.html) in directives such as `@module`, `@name` and `@alias`, `valid-jsdoc` rule throws error: "JSDoc syntax error"

``` javascript
/** @constructor */
function Test() {
}

/**
 * @alias Test#test
 * @returns {void}
 * */
Test.prototype.test = function () {
};
```

In the example above `Test#test` namepath fails, whereas `Test.test` works.
